src/utils/compileJsx.ts

```diff
@@ -1,11 +1,22 @@
 import { transform } from '@babel/standalone';
 
-export const openFragmentTag = '<React.Fragment>';
-export const closeFragmentTag = '</React.Fragment>';
+export const ReactFragmentPragma = 'R_F';
+export const ReactCreateElementPragma = 'R_cE';
+
+export const openFragmentTag = '<>';
+export const closeFragmentTag = '</>';
```

Comment:
Might need to be careful here. While this hack allows us to format the code, we are forced to unwrap it in a number of places (like [here]). I think the point of this variable is we can change it now, but might just have to manually test drive a few features to be sure.

[here]: https://github.com/seek-oss/playroom/blob/74be8e38e9f91d14c7085a1382051b25772b87a1/src/Playroom/CodeEditor/CodeEditor.tsx#L58-L66

Reply:
I've added some unit tests https://github.com/seek-oss/playroom/pull/304/commits/2e889ebaf5f241056c7738c94ad43574cddb61bb